main/cron/remind_course_expiration.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * Course expiration reminder.
+ * @package chamilo.cron
+ * @author Imanol Losada <imanol.losada@beeznest.com>
+ */
+require_once __DIR__ . '/../inc/global.inc.php';
+
+/**
+ * Initialization
+ */
+if (php_sapi_name() != 'cli') {
+    exit; //do not run from browser
+}
+
+// Days before expiration date to send reminders
+define("OFFSET", 2);
+$today = date("Y-m-d");
```

Comment:
Although date() will use the server's date, this is not necessarily UTC (the server date might be UTC+2, but the database stores times in UTC+0), so you should always use the api_get_utc_datetime() (or at least gmdate()).